routes/users.go

```diff
@@ -3,15 +3,28 @@ package routes
 import (
 	"net/http"
 
+	"github.com/Real-Dev-Squad/wisee-backend/models"
 	"github.com/gin-gonic/gin"
+	"github.com/uptrace/bun"
 )
 
-func userRoutes(rg *gin.RouterGroup){
-	users :=  rg.Group("/users")
+func userRoutes(rg *gin.RouterGroup, db *bun.DB) {
+	users := rg.Group("/users")
 
-	users.GET("", func(c * gin.Context){
-		c.JSON(http.StatusOK, gin.H{
-			"message": "users",
-	})
+	users.GET("", func(ctx *gin.Context) {
+
+		var users []models.User
+		err := db.NewSelect().Model(&users).OrderExpr("id ASC").Limit(10).Scan(ctx)
```

Comment:
users.GET() function. The OrderExpr() method takes a string as its argument, but you are passing a struct literal (models.User). This will not work, because the OrderExpr() method expects a string that represents the column name to order by.